.github/workflows/healthchecks_influxdb_ci.yml

```diff
@@ -52,7 +52,7 @@ jobs:
       - name: Setup InfluxDB
         uses: maikebing/influxdb-action@v5
         with:
-          influxdb_version: latest
+          influxdb_version: 2.7.1

```

Comment:
latest was hanging (https://github.com/influxdata/influxdb-action/issues/5), so I've used the 2.7.1 as it's the stable release according to https://en.wikipedia.org/wiki/InfluxDB

Reply:
Thanks.
https://github.com/Xabaril/AspNetCore.Diagnostics.HealthChecks/pull/2110